frontend/packages/integration-tests-cypress/tests/crud/k8-openshift-cruds.cyspec.ts

```diff
@@ -0,0 +1,190 @@
+/* eslint-disable no-restricted-globals, promise/catch-or-return, promise/no-nesting */
+import { OrderedMap } from 'immutable';
+import { safeLoad, safeDump } from 'js-yaml';
+import * as _ from 'lodash';
+
+import { testName, editHumanizedKind, deleteHumanizedKind } from '../../support';
+import { listPage } from '../../views/list-page';
+import { detailsPage } from '../../views/details-page';
+import { modal } from '../../views/modal';
+import * as yamlEditor from '../../views/yaml-editor';
+import { errorMessage } from '../../views/form';
+
+describe('Kubernetes resource CRUD operations', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+  });
+
+  // after() defined at end of file
+
+  const k8sObjs = OrderedMap<string, { kind: string; namespaced?: boolean }>()
+    .set('pods', { kind: 'Pod' })
+    .set('services', { kind: 'Service' })
+    .set('serviceaccounts', { kind: 'ServiceAccount' })
+    .set('secrets', { kind: 'Secret' })
+    .set('configmaps', { kind: 'ConfigMap' })
+    .set('persistentvolumes', { kind: 'PersistentVolume', namespaced: false })
+    .set('storageclasses', { kind: 'StorageClass', namespaced: false })
+    .set('ingresses', { kind: 'Ingress' })
+    .set('cronjobs', { kind: 'CronJob' })
+    .set('jobs', { kind: 'Job' })
+    .set('daemonsets', { kind: 'DaemonSet' })
+    .set('deployments', { kind: 'Deployment' })
+    .set('replicasets', { kind: 'ReplicaSet' })
+    .set('replicationcontrollers', { kind: 'ReplicationController' })
+    .set('persistentvolumeclaims', { kind: 'PersistentVolumeClaim' })
+    .set('statefulsets', { kind: 'StatefulSet' })
+    .set('resourcequotas', { kind: 'ResourceQuota' })
+    .set('limitranges', { kind: 'LimitRange' })
+    .set('horizontalpodautoscalers', { kind: 'HorizontalPodAutoscaler' })
+    .set('networkpolicies', { kind: 'NetworkPolicy' })
+    .set('roles', { kind: 'Role' });
+  const openshiftObjs = OrderedMap<string, { kind: string; namespaced?: boolean }>()
+    .set('deploymentconfigs', { kind: 'DeploymentConfig' })
+    .set('buildconfigs', { kind: 'BuildConfig' })
+    .set('imagestreams', { kind: 'ImageStream' })
+    .set('routes', { kind: 'Route' })
+    .set('user.openshift.io~v1~Group', { kind: 'user.openshift.io~v1~Group', namespaced: false });
+  const serviceCatalogObjs = OrderedMap<string, { kind: string; namespaced?: boolean }>().set(
+    'clusterservicebrokers',
+    {
+      kind: 'servicecatalog.k8s.io~v1beta1~ClusterServiceBroker',
+      namespaced: false,
+    },
+  );
+
+  let testObjs = Cypress.env('openshift') === true ? k8sObjs.merge(openshiftObjs) : k8sObjs;
```

Comment:
We should default to OpenShfit and only skip those objects if a different env var is set.

Reply:
irt  '...skip those objects if a different env var is set', Are you suggesting a new env var such as 'skipOpenShift=true' which would then bypass  `openshiftObjs` ? -thanks
Wouldn't that just be the reverse of what we have now, like:
`let testObjs = Cypress.env('skipOpenShift') === true ? k8sObjs : k8sObjs.merge(openshiftObjs);`  -thanks